### PR TITLE
created scripts to create files with records, added script in package…

### PR DESCRIPTION
**Pull request by leungnamson1023**
… that runs all files at once

https://trello.com/c/BEIQzqei/7-creating-script-to-run-data-generation-medium

### CODE REVIEW

File: db/batchSeed/address/createBatch.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const faker = require('faker');
+
+/***** schema
```
Comment (ZeMunchkin) [inline db/batchSeed/address/createBatch.js]: I wouldn't merge with commented out code -- master should only have clean working code

File: db/batchSeed/restaurantBatch/createBatch.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  SmallBites: 10,
+  Salads: 11,
+  WineBars: 12,
+  Seafood: 13,
+  Burmese: 14,
+};
+
+const food = Object.keys(types);
+
+// let batchSize = 10000000;
```
Comment (ZeMunchkin) [inline db/batchSeed/restaurantBatch/createBatch.js]: Same thing here. I'd take this out

File: db/batchSeed/restaurant_typeBatch/createBatch.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const faker = require('faker');
+
+/***** structure
```
Comment (ZeMunchkin) [inline db/batchSeed/restaurant_typeBatch/createBatch.js]: Same

File: db/batchSeed/types/createBatch.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const data = require('../../data/MomaDummyData.js');
+
+const typesArr = Object.keys(data.types);
+
+// const createFile = () => {
```
Comment (ZeMunchkin) [inline db/batchSeed/types/createBatch.js]: Same thing

File: db/batchSeed/address/createBatch.js (hunk 1 above)
Comment (leungnamson1023) [reply db/batchSeed/address/createBatch.js]: got it! will remove all commented out code here and below as well

**Comment by leungnamson1023** [comment]
will remove all commented code! thanks for the review

**Pull request merged**